### PR TITLE
feat(files): music_disc_redstone_levels

### PR DESCRIPTION
**Pull request by SquatchHunter**
Gives all Music Discs a redstone power level indicator in the tooltip.
Addresses: #36 
By checking the following boxes with an X, you ensure that:

- [x] The pack was tested ingame in at least one device.
- [x] The pack is an existing BT pack, is a missing pack from VT or is an accepted pack/change in a discussion.
- [x] The pack code follows the style guide.
- [x] The commits follow the contribution guidelines.
- [x] The PR follows the contribution guidelines.

- [x] (Optional) Tested in Windows
- [ ] (Optional) Tested in Android
- [ ] (Optional) Tested in iOS
- [ ] (Optional) Tested in any console
- [ ] (Optional) Tested in BDS

image

### CODE REVIEW

File: resource_packs/files/utility/music_disc_redstone_level/font/glyph_E1.png
Comment (drav0011) [inline resource_packs/files/utility/music_disc_redstone_level/font/glyph_E1.png:1]: This pack will be incompatible with hunger preview as both use the same glyph file
Move redstone icon to 3rd position, remove extra greenlines, and use latest vanilla glyph files (missing vanilla camera icon)
Add compatibility file for hunger preview + music disc redstone level
The compatibility pack should include all changes from both packs

Note: Hunger preview is also not be using the latest glyph file, forgot to update that, fixing that also in this PR would be aprecciated

Vanilla glyph_E1.png
image

File: resource_packs/packs.json [hunk 1]
```diff
@@ -649,6 +649,11 @@
 					"name": "Mine Progress Bar",
 					"description": "Changes the breaking texture of blocks to a clean ring around the outside."
 				},
+				{
+					"id":"music_disc_redstone_level",
+					"name":"Music Disc Redstone Level",
+					"description":"Gives all Music Discs a redstone power level indicator in the tooltip."
```
Comment (drav0011) [inline resource_packs/packs.json:655]: Pack should be after hunger preview

**Review by drav0011 — CHANGES_REQUESTED** [review]

File: resource_packs/files/utility/music_disc_redstone_level/font/glyph_E1.png
Comment (SquatchHunter) [reply resource_packs/files/utility/music_disc_redstone_level/font/glyph_E1.png:1]: didnt know that, all sources I saw still list the old E1 image
will update, also forgot to check compatabilities so will also make that, shouldnt hunger preview just use the already existing icon though?

File: resource_packs/packs.json (hunk 1 above)
Comment (SquatchHunter) [reply resource_packs/packs.json:655]: why? I thought it was just alphabetical and VSC lists its alphabetically here

**Review by drav0011 — CHANGES_REQUESTED** [review]
The combination pack is incorrect
It should be the same as a standalone pack, when 2 packs are combined, those packs are not added to the mcpack, instead the combination is added

So this actual combination is missing the text files and the image should be in the font directory

**Review by drav0011 — APPROVED** [review]
lgtm

**Pull request merged**